### PR TITLE
Peribolos: Revert changes from PR 4363

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
AI from the recent Peribolos Incident - #4365 

---

The PR reverts the following commits from the PR #4363 :

- 966e7360f3918c0f06599c0829498b5a204ed6af (update sig-contributor-experience members)
- 501659d66a6d068829e45200888d6444e3ceb243 (update sig-release & release-team team perms)
- 1780f18adb5607c272396df399572bcc89dd5ce5 (update sig-testing team perms)
- e8c0043130bcb8fdc723cbdcad7566997909336e (update sig-k8s-infra team perms)
- 2c00c6b23242dfe8940d8a5529be563578535d8c (update production-readiness team perms)
- 4b36d60bad339107cc6a31ee5688a583dd80e34c (update sig-cloud-provider team perms)
- dcfce93dc605e8847a642a4455f8ebdc69fa77d2 (verified cluster-api-provider-azure-* team perms)

Note - following commit was reverted previously with PR #4364 

- eda81258d445e6d8d3968623970a96dcafe52079 (update cluster-api-provider-vsphere-* team perms)

---

TODO: (after `--skip-removals` functionality is removed from Peribolos altogether)

- Revert 7094fcb1c8ae097165d13e2385fd067b9fbf2aaf (update admin/update.sh to include --skip-removals flag)

/assign @MadhavJivrajani 
/sig contributor-experience

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml
Comment (MadhavJivrajani) [inline config/kubernetes/sig-release/teams.yaml:325]: katcosgrove is listed a docs shadow [1], is this line still accurate?
[1] https://github.com/kubernetes/sig-release/blob/master/releases/release-1.28/release-team.md

**Review by MadhavJivrajani — COMMENTED** [review]
❤️ 
/lgtm 
/approve 
/hold 

Feel free to remove hold if my comment requires no change to be made, else will lgtm again post change.

**Comment by MadhavJivrajani** [comment]
> TODO: (after --skip-removals functionality is removed from Peribolos altogether)

We should remove flag before functionality ^

File: config/kubernetes/sig-release/teams.yaml
Comment (Priyankasaggu11929) [reply config/kubernetes/sig-release/teams.yaml]: @MadhavJivrajani, fixed in latest commit https://github.com/kubernetes/org/pull/4374/commits/d54330b68455ba4410ad4507c7f4e2a93ee3308e . Thanks!

**Comment by Priyankasaggu11929** [comment]
> > TODO: (after --skip-removals functionality is removed from Peribolos altogether)
> 
> We should remove flag from admin script before functionality ^

@MadhavJivrajani, thinking of doing ^ in a separate PR then.

Just trying to sync the timings of both following changes:
-  `k/test-infra` PR to remove the --skip-removals  flag functionality
- `k/org` PR to revert  https://github.com/kubernetes/org/commit/7094fcb1c8ae097165d13e2385fd067b9fbf2aaf (update admin/update.sh to include --skip-removals flag)

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm
/hold cancel

**Pull request merged**
